kinto/__main__.py

```diff
@@ -222,6 +238,11 @@ def main(args=None):
         env = bootstrap(config_file, options={"command": "delete-collection"})
         return scripts.delete_collection(env, parsed_args["bucket"], parsed_args["collection"])
 
+    elif which_command == "flush":
+        #env = bootstrap(config_file, option={"command": "flush-cache"})
+        if parsed_args["flush-cache"]:
+            return cache.flush()
```

Comment:
I guess we don't need both a command and a subparser.

Reply:
i was thinking maybe if in the future someone needed to add another flush functionality,they could simply add a subparser underneath this. 

I will refactor this asap with just flush-cache

Reply:
The cache function has got a implementation per backend. Here we are using the one defined in the `cache/__init__.py` interface. It is the responsibility of each backend to implement it.

Reply:
All tests passes, the only thing is that there are no tests yet for the code you wrote:

```/home/travis/build/Kinto/kinto/kinto/__main__.py                                127      2     52      1    97%   243-244, 241->243```